### PR TITLE
dev-python/python-evdev: update to 1.3.0 + new proxy maintainer

### PR DESCRIPTION
**Pull request by epsilon-0**
Also add support for python 3.8

Closes: https://bugs.gentoo.org/720780

Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: dev-python/python-evdev/python-evdev-1.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (juippis) [inline dev-python/python-evdev/python-evdev-1.3.0.ebuild]: One empty line below EAPI.

File: dev-python/python-evdev/python-evdev-1.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+DESCRIPTION="Python library for evdev bindings"
+HOMEPAGE="https://python-evdev.readthedocs.org/"
+SRC_URI="https://github.com/gvalkov/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
```
Comment (juippis) [inline dev-python/python-evdev/python-evdev-1.3.0.ebuild]: Handled via *distutils-r1.eclass*.

File: dev-python/python-evdev/python-evdev-1.3.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Python library for evdev bindings"
+HOMEPAGE="https://python-evdev.readthedocs.org/"
+SRC_URI="https://github.com/gvalkov/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
```
Comment (juippis) [inline dev-python/python-evdev/python-evdev-1.3.0.ebuild]: Empty variable can be dropped.

**Comment by epsilon-0** [comment]
> I see upstream has python test files. Why aren't the tests enabled in the ebuild? It would immediately tell if python3.8 works or not.

Like I haven't found how to enable the tests, also they are like 5 years old...

Will search some more

File: dev-python/python-evdev/python-evdev-1.3.0.ebuild (hunk 1 above)
Comment (epsilon-0) [reply dev-python/python-evdev/python-evdev-1.3.0.ebuild]: done

File: dev-python/python-evdev/python-evdev-1.3.0.ebuild (hunk 2 above)
Comment (epsilon-0) [reply dev-python/python-evdev/python-evdev-1.3.0.ebuild]: done

File: dev-python/python-evdev/python-evdev-1.3.0.ebuild (hunk 3 above)
Comment (epsilon-0) [reply dev-python/python-evdev/python-evdev-1.3.0.ebuild]: done

**Comment by epsilon-0** [comment]
> I see upstream has python test files. Why aren't the tests enabled in the ebuild? It would immediately tell if python3.8 works or not.

Added the tests and still keeping the python3_8 because it is working so far on my system :smile_cat:

Unfortunately the tests are a bit old, I've made a issue on the upstream to check if the tests should still be supported https://github.com/gvalkov/python-evdev/issues/127

Until then I think this looks fine.

**Pull request closed without merging**